soroban-env-host/Cargo.toml

```diff
@@ -23,7 +23,8 @@ static_assertions = "=1.1.0"
 sha2 = "=0.10.8"
 hex-literal = "=0.4.1"
 hmac = "=0.12.1"
-ed25519-dalek = {version = "=2.0.0", features = ["rand_core"] }
+# NB: We'll need to pin this again after switching the Core to the new env version.
```

Comment:
Do we need an issue to track this?

Reply:
Good point, opened https://github.com/orgs/stellar/projects/39/views/3?filterQuery=no%3Astatus+pin&pane=issue&itemId=68347479